src/containers/Flow/FlowList/FlowList.tsx

```diff
@@ -202,6 +202,30 @@ export const FlowList = () => {
     },
   };
 
+  const inputSxStyle = {
```

Comment:
Let's move this to Autocomplete component and make it configurable using a prop like. isFilterType: boolean

if isFilterType- apply these styles.